src/components/textbox/textbox.pw.tsx

```diff
@@ -247,47 +241,40 @@ test.describe("Prop checks for Textbox component", () => {
     TextboxProps["inputWidth"],
     number,
     number
-  ][]).forEach(([labelVals, inputVals, labelRatio, inputRatio]) => {
-    test(`should render with correct ratios when inputWidth prop is ${inputVals} and labelWidth prop is ${labelVals}`, async ({
+  ][]).forEach(([labelVal, inputVal, labelRatio, inputRatio]) => {
+    test(`should render with correct ratios when inputWidth prop is ${inputVal} and labelWidth prop is ${labelVal}`, async ({
       mount,
       page,
     }) => {
       await mount(
         <TextboxComponent
           labelInline
-          labelWidth={labelVals}
-          inputWidth={inputVals}
+          labelWidth={labelVal}
+          inputWidth={inputVal}
         />
       );
 
       const label = getDataElementByValue(page, "label");
       const labelParent = await label.evaluateHandle((element: Element) => {
         return element.parentElement;
       });
-
-      /* Had to create my own textbox locator here instead of using textbox(page), as that seemed to encounter
-      issues with our getStyle helper, potentially due to how it handles roles */
-
-      const textboxElement = await textboxInput(page).evaluateHandle(
+      const textboxJsHandle = await textbox(page).evaluateHandle(
         (element: Element) => {
-          return element.parentElement;
+          return element;
         }
       );
 
-      expect(
-        assertCssValueIsApproximately(
-          (labelParent as unknown) as Locator,
-          "width",
-          labelRatio
-        )
+      const labelRatioValue = await page.evaluate(
+        (element) => window.getComputedStyle(element).getPropertyValue("width"),
+        labelParent
       );
-      expect(
-        assertCssValueIsApproximately(
-          (textboxElement as unknown) as Locator,
-          "width",
-          inputRatio
-        )
+      const inputRatioValue = await page.evaluate(
+        (element) => window.getComputedStyle(element).getPropertyValue("width"),
+        textboxJsHandle
       );
+
+      expect(parseInt(labelRatioValue)).toBeGreaterThanOrEqual(labelRatio - 2);
```

Comment:
**question** (non-blocking): are we sure we need to allow a window of 2 either side of the actual value? I know we can't guarantee it being bang on but this seems a wider range than I'm comfortable with. Have we tried it with a gap of say 1, or even 0.5, on either side?

Reply:
Yeah I don't object to making this a little stricter